python/segyio/open.py

```diff
@@ -80,12 +80,16 @@ def open(filename, mode="r", iline = 189,
         File access mode, read-only ('r', default) or read-write ('r+')
 
     iline : int or segyio.TraceField
-        Inline number field in the trace headers. Defaults to 189 as per the
-        SEG-Y rev1 specification
+        Overrides inline field offset in the trace headers. If no value is set,
+        value defined in xml mapping or standard-defined 189 would be used. By
+        setting this value user takes over responsibility of assuring parsing
+        can be done correctly.
 
     xline : int or segyio.TraceField
-        Crossline number field in the trace headers. Defaults to 193 as per the
-        SEG-Y rev1 specification
+        Overrides crossline field offset in the trace headers.  If no value is
+        set, value defined in xml mapping or standard-defined 193 would be used.
+        By setting this value user takes over responsibility of assuring parsing
+        can be done correctly.
```

Comment:
Just a reminder that if we update the description for the `iline` parameter, we should update this accordingly.